### DISCUSSION TITLE
DXF Viewer

### DISCUSSION THREAD

**Discussion by BigAloz** [haplokuon/netDxf#451] [General]
Hello

I have just download netDXF it looks very promising.

I was wondering if anybody has written a viewer using nextDXF in C# ? and whether they are willing to share.

Regards,

Alan Buchanan

**Comment by BigAloz** [comment]
I have started (and all most completed a viewer in Windows.

Turns out it is not that difficult to generate .Graphics commands from the netDXF library.

If anybody is interested I can share. Still a work in progress.

**Comment by xaotix** [comment]
I have a very simple that creates in Canvas WPF, only for preview.

**Reply by BigAloz** [reply]
That would be great if you share

I have written a reasonable viewer using e.graphics library 

actually wasn’t that difficult

**Reply by xaotix** [reply]
https://github.com/xaotix/DLM.Extensions

**Reply by DavidWishengrad** [reply]
Thank you. I didn't have the time to figure out how to run your code. I just read it in notepad. I will give it a look in the future when I have more time. Portuguese?

**Reply by xaotix** [reply]
Yeah, I'm from Brazil.
